readme.md

```diff
@@ -635,6 +637,49 @@ got('https://api.github.com/some-endpoint', {
 });
 ```
 
+##### \_pagination
+
+Type: `object`
+
+**Note:** This feature is marked as experimental as we're [looking for feedback](https://github.com/sindresorhus/got/issues/1052) on the API and how it works. The feature itself is stable, but the API may change based on feedback. So if you decide to try it out, we suggest locking down the `got` dependency semver range or use a lockfile.
+
+###### \_pagination.transform
+
+Type: `Function`\
+Default: `response => JSON.parse(response.body)`
+
+A function that transform [`Response`](#response) into an array of items. This is where you should do the parsing.
+
+###### \_pagination.paginate
+
+Type: `Function`\
+Default: [`Link` header logic](source/index.ts)
```

Comment:
I think we should explain the Link header logic in English prose, not just link to the source.